src/lib/plugin.js

```diff
@@ -308,17 +319,23 @@ class FiveBellsLedger extends EventEmitter2 {
         })
 
         if (res.statusCode !== 200) {
+          this.emit('reject', transfer,
+            new Buffer('Unexpected status code: ' + res.statusCode))
           throw new Error('Unexpected status code: ' + res.statusCode)
```

Comment:
Then you need to wrap the `request()` in a try/catch as well -- connection errors will still throw.